libs/ballot-interpreter/src/bubble-ballot-rust/timing_marks/corners/border_finding.rs

```diff
@@ -217,13 +239,15 @@ impl GridBorder {
 pub struct Options {
     pub maximum_vertical_timing_mark_center_distance_error_ratio: f32,
     pub maximum_horizontal_timing_mark_center_distance_error_ratio: f32,
+    pub min_border_timing_mark_score: UnitIntervalScore,
 }
 
 impl DefaultForGeometry for Options {
     fn default_for_geometry(_geometry: &Geometry) -> Self {
         Self {
             maximum_vertical_timing_mark_center_distance_error_ratio: 0.5,
             maximum_horizontal_timing_mark_center_distance_error_ratio: 0.5,
+            min_border_timing_mark_score: UnitIntervalScore(0.5),
```

Comment:
I'm a bit surprised that you didn't go with a higher threshold for these marks. I understand that we don't want to reject too aggressively, but I have an intuition that this scoring threshold won't reject anything that makes it through the dimension check.

Reply:
Given that we're going to be doing a decent amount of QA in the near future, I'll try putting this at 80%. We can bump it down if needed.